### DISCUSSION TITLE
Feature Request: Postal Has Buttons Now

### DISCUSSION THREAD

**Discussion by PaPi-Mario** [SFX-WoW/Masque#150] [Issues] [answered]
Postal have gotten buttons now, needs to be skinned pleaz!

https://www.curseforge.com/wow/addons/postal

**Comment by StormFX** [comment, marked as answer]
Thanks for the suggestion. However, you'll need to request this feature on Postal's project page. Masque operates as a library, which means that add-ons have to provide support for Masque, not the other way around.
